### DISCUSSION TITLE
Multi-site-per-tenant in multitenant setup?

### DISCUSSION THREAD

**Discussion by sturlath** [OrchardCMS/OrchardCore#15309] [General]
Hopefully you will understand this question bit just let me know if not.

Is it possible with OrchardCMS to allow new tenants to sign up and then create many sub pages/spaces where they manage all content? 

Each sub page/space needs to have its own subdomain.

So we would have
* 1 admin site/portal (us the company)
* Many tenant portals that can have 1 or more product sites that they control. Each site would have different look and feel and advertise a different product and have a different domain URL

Is this built in or does this need custom work? Has anybody done it before?

**Comment by hishamco** [comment]
- The sign up (registration) is feature that could be enable or disable for each tenant
- Each tenant its own subdomain.
- admin site could be the default tenant
- Each tenant has it's own content items and theme

**Reply by sturlath** [reply]
Brilliant thank you! This will then work in my case. Been watching videos and will now create a project to play with.

**Reply by hishamco** [reply]
All the best
